### ISSUE TITLE
Unable to sign in

### ISSUE THREAD

**Issue by guillaumechauvat** [Ultimaker/Cura#10252]
### Application Version

4.10.0

### Platform

Arch linux

### Printer

Ultimaker 2

### Reproduction steps

1. Launch Cura with no data saved
2. Click on "sign in" when the menu appears
3. Get redirected to Firefox with a login page and sign in there.

### Actual results

The login appears successful in Firefox, but Cura doesn't see it.

![Screenshot_20210808_110142](https://user-images.githubusercontent.com/28393599/128626869-d8ed78e3-5037-467b-945c-d3e77c8cec68.png)

![Screenshot_20210808_110044](https://user-images.githubusercontent.com/28393599/128626951-3c1d4640-4b11-4438-b91e-825dec091c9e.png)

Clicking on "sign in" again in Cura has no effect.

If I skip the login there, I can click on "Check for account updates" but it fails and looks like this:

![Screenshot_20210808_110126](https://user-images.githubusercontent.com/28393599/128626965-b7cfe9ec-9058-411d-b9f7-e414e5282e78.png)

### Expected results

I can sign in and synchronise my account.

### Checklist of files to include

- [X] Log file
- [ ] Project file

### Additional information & file uploads

Logs: 
[cura.log](https://github.com/Ultimaker/Cura/files/6950476/cura.log)

**Comment by fvrmr** [comment]
Hi @guillaumechauvat did this happen with the AppImage of Cura?

**Comment by guillaumechauvat** [comment]
No, it's the [Arch Linux package](https://archlinux.org/packages/community/any/cura/). I can try installing another way if you want.

**Comment by guillaumechauvat** [comment]
Okay, so I tried using the AppImage and it gave some clues about what is going on.

I started launching but crashed with
```
ModuleNotFoundError: No module named 'keyring.backends.null'
```

It turns out that I had a file `$HOME/.config/python_keyring/keyringrc.cfg` that contained:
```
[backend]
default-keyring=keyring.backends.null.Keyring
```

After removing that file, the AppImage launched. It failed in a different way when I clicked on "sign in":
```
/usr/bin/xdg-open: line 613: 595893 Segmentation fault      (core dumped) kde-open${KDE_SESSION_VERSION} "$1"
```

However now the Cura version installed from the Arch Linux package manager works fine!

**Comment by fvrmr** [comment]
Thank you for taking a look at it. I will bring this up for discussion with the team. Keep you posted!

**Comment by konskarm** [comment]
That was a really helpful and detailed report @guillaumechauvat! We do not support the Keyring on Linux (since we'd have to ship a specific keyring package with it) and it wasn't explicitly ignoring local Keyring configs. I've fixed it now to make sure that it won't load any keyring backends on Linux https://github.com/Ultimaker/Cura/pull/10302. 
For our internal tracking, devs see ticket CURA-8490.

**Comment by Ghostkeeper** [comment]
This is fixed by #10302 for Cura 4.11.
